crates/wasmi/src/store.rs

```diff
@@ -714,6 +742,62 @@ impl<T> Store<T> {
         self.data
     }
 
+    pub fn limiter(
```

Comment:
I know it might seem annoying but for the sake of consistency I like to see doc comments even on non-public APIs. Same with the other methods below. Short description and maybe rational is enough.

Reply:
Oh yeah sorry missed this one! Fixed.

Reply:
Pushed the fix already? :)
This PR still needs resolution for the CI.